website/docs/en/config/infrastructure-logging.mdx

```diff
@@ -0,0 +1,126 @@
+import WebpackLicense from '@components/webpack-license';
+
+<WebpackLicense from="https://webpack.js.org/configuration/infrastructureLogging/" />
+
+# InfrastructureLogging
+
+Options for infrastructure level logging.
```

Comment:
Can we explain what is `infrastructure level logging`? I think this concept is not very clear